converters/fromZigbee.js

```diff
@@ -4039,7 +4039,7 @@ const converters = {
             const presetLookup = {0: 'programming', 1: 'manual', 2: 'temporary_manual', 3: 'holiday'};
             switch (dp) {
             case tuya.dataPoints.moesSsystemMode:
-                return {preset: presetLookup[value]};
+                return {preset: presetLookup[value], system_mode: 'heat'};
```

Comment:
Is there some proper way to do this, without failing the test?

I could also map programming -> auto, but it is not quite correct, ZCL says "auto" is about heat vs cool, not the schedule (although Home Assistant does interpret auto as schedule, apparently).

Reply:
The reason the tests fails is because of the linting (line too long). check: https://github.com/Koenkk/zigbee-herdsman-converters/actions/runs/3321185565/jobs/5488487468

Reply:
It turns out, I was completely misinterpreting the test.